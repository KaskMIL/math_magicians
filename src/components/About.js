import React from 'react';
import Header from './Header';

function About() {
  return (
    <>
      <Header />
      <article>
        <p>
          Mathematics is not about numbers, equation, computations or alogithms:
          It is about understanding.-William Paul Thurston
        </p>
      </article>
    </>
  );
}

export default About;
