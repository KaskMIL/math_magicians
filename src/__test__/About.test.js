import React from "react";
import renderer from "react-test-renderer"
import { BrowserRouter } from 'react-router-dom';
import About from "../components/About";

test('Should render About correctly', () => {
  const tree = renderer.create(
    <BrowserRouter>
      <About />
    </BrowserRouter>
  ).toJSON();
  expect(tree).toMatchSnapshot();
})
