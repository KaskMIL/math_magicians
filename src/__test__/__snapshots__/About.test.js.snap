// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render About correctly 1`] = `
Array [
  <header
    className="header"
  >
    <h1>
      Magic Magicians
    </h1>
    <nav>
      <ul>
        <li>
          <a
            className="link"
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
        <li>
          <a
            className="link"
            href="/calculator"
            onClick={[Function]}
          >
            Calculator
          </a>
        </li>
        <li>
          <a
            className="link"
            href="/about"
            onClick={[Function]}
          >
            About
          </a>
        </li>
      </ul>
    </nav>
  </header>,
  <article
    className="about"
  >
    <p>
      Mathematics is not about numbers, equation, computations or alogithms: It is about understanding.-William Paul Thurston
    </p>
  </article>,
]
`;
